sc/social/like/plugins/facebook/browser.py

```diff
@@ -62,13 +58,13 @@ def is_plone_5(self):
 
     def fbjs(self):
         js_source = """
-    (function() {{
-        var po = document.createElement('script');
-        po.async = true;
-        po.src = document.location.protocol + '//connect.facebook.net/{0}/all.js#xfbml=1';
-        var head = document.getElementsByTagName('head')[0];
-        head.appendChild(po);
-    }}());
+    (function(d, s, id) {{
+      var js, fjs = d.getElementsByTagName(s)[0];
+      if (d.getElementById(id)) return;
+      js = d.createElement(s); js.id = id;
+      js.src = "//connect.facebook.net/{0}/sdk.js#xfbml=1&version=v2.10";
```

Comment:
This example from Facebook API is broken, it miss the `&version` part and then asks for the version at first run (I think it is related with locale change this error)